packages/dbase4/dbase4.0.1.5/opam

```diff
@@ -0,0 +1,44 @@
+# This file is generated by dune, edit dune-project instead
+opam-version: "2.0"
+synopsis: "A DBF/DBT files reader library"
+description:
+  "A DBF reader library with MEMO support and sequental record access"
+maintainer: ["Mykhayl Puzanov <pum@example.com>"]
+authors: ["Mykhayl Puzanov <pum@example.com>"]
+license: "GPL-3.0-or-later"
+tags: ["xBASE" "DBF" "data access"]
+homepage: "https://github.com/hothing/ocaml-libdbf"
+bug-reports: "https://github.com/hothing/ocaml-libdbf/issues"
+depends: [
+  "dune" {>= "3.17"}
+  "ocaml" {>= "5.2"}
+  "cstruct"
+  "ppx_cstruct"
+  "bitstring"
+  "ppx_bitstring"
+  "odoc" {= with-doc}
+  "alcotest" {= with-test}
```

Comment:
```suggestion
  "odoc" {with-doc}
  "alcotest" {with-test}
```

Reply:
hi @hothing 

this error caused opam to essentially fail on the CI

it seems to come from your dune-project file so I'd recommend to fix it upstream